src/application-error-boundary/ApplicationErrorBoundary.jsx

```diff
@@ -72,6 +72,9 @@ class ApplicationErrorBoundary extends React.Component {
      * next update occurs.
      */
     this.errorRef.current = error;
+    if (error && window.newrelic) {
+      window.newrelic.noticeError(error);
```

Comment:
Is there any way to test this?

Reply:
I was able to list the errors in new relic by running in production mode locally along with new-relic license key and terra-ui/terra-dev-site's application ID. The errors were registered for both the repos.

[Here](https://rpm.newrelic.com/accounts/1721539/browser/142381315/errors#?groupBy=errorMessage&tw_start=&tw_end=now&tw_offset=259200000) is a link to the registered JS errors in new-relic for terra-ui. If you are unable to see the errors change the time-picker to last 7 days. Errors that say "whoops" and "This is an error test"  was simulated locally in production mode.